data/json/items/comestibles/cereal.json

```
@@ -0,0 +1,158 @@
+[
+  {
+    "abstract": "cereal_tpl",
+    "type": "COMESTIBLE",
+    "name": { "str_sp": "cereal" },
+    "weight": "40 g",
+    "volume": "250 ml",
+    "color": "yellow",
+    "symbol": "%",
+    "spoils_in": "90 days",
+    "container": "box_small",
+    "comestible_type": "FOOD",
+    "quench": -2,
+    "calories": 145,
+    "healthy": 1,
+    "description": "A generic box of cereal, you shouldn't see this.",
+    "price": 300,
+    "price_postapoc": 200,
+    "material": "wheat",
+    "flags": [ "EDIBLE_FROZEN", "USE_EAT_VERB" ],
+    "looks_like": "cereal",
+    "vitamins": [ [ "vitA", 10 ], [ "vitC", 7 ], [ "calcium", 7 ], [ "iron", 32 ] ],
+    "fun": 8
+  },
+  {
+    "abstract": "cereal_fpsugar_tpl",
+    "type": "COMESTIBLE",
+    "name": { "str_sp": "FoodPlace cereal" },
+    "quench": -3,
+    "calories": 225,
+    "healthy": -1,
+    "description": "A generic box of FoodPlace brand sugary cereal, you shouldn't see this.",
+    "price_postapoc": 400,
+    "charges": 6,
+    "material": "foodplace_foodstuff",
+    "looks_like": "cereal",
+    "vitamins": [ [ "vitA", 5 ], [ "vitC", 2 ], [ "calcium", 2 ], [ "iron", 16 ] ],
+    "fun": 11,
+    "addiction_potential": 5,
+    "addiction_type": "caffeine",
+    "fatigue_mod": 25,
+    "stim": 10,
+    "copy-from": "cereal_tpl"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "cereal_fp1",
+    "name": { "str_sp": "Snicker-Snacks cereal" },
+    "description": "FoodPlace brand Snicker-Snack cereal.  Each tiny \"Snicker-Snack\" is shaped like an earth food!",
```

```suggestion
    "description": "FoodPlace brand Snicker-Snack cereal.  Each tiny \"Snicker-Snack\" is shaped like human food!",
```
Not sure about this one `an earth food` is pretty funny but I'm afraid it kind of implies Foodplace is controled by aliens.